sisyphus.pl

```diff
@@ -1097,6 +1109,42 @@ =head1 DESCRIPTION
     echo OK
 }
 
+# Start up the analysis in the ngi pipeline for 
+# this flowcell
+startNGIPipeline () {
+
+    remoteHost="";
+    remotePort="";
+    runfolderName="";
+
+    if [ -z "\$1" ]
+    then
+      echo "-remote host #1 is zero length or empty."
+      exit 1;
+    else
+       remoteHost=\$1;
+    fi
+
+    if [ -z "\$2" ]
+    then
+      echo "-remote port #2 is zero length or empty."
+      exit 1;
+    else
+       remotePort=\$2;
+    fi
+    
+    if [ -z "\$3" ]
+    then
+      echo "-runfolder name path #3 is zero length or empty."
+      exit 1;
+    else
+       runfolderName=\$3;
+    fi
+    
+    ssh \$remoteHost "curl \$remoteHost:\$remotePort/flowcell_analysis/runfolderName"
```

Comment:
should the host in the curl command perhaps be localhost or will it resolve to that on the remote anyway?